lib/sequel/extensions/seek_pagination.rb

```diff
@@ -5,28 +5,19 @@ module Sequel
   module SeekPagination
     class Error < StandardError; end
 
-    def seek_paginate(count, from: nil, after: nil, from_pk: nil, after_pk: nil, not_null: nil)
```

Comment:
- <code>count</code> is removed because it's redundant with Sequel <code>limit</code>
- <code>from</code>, <code>after</code>, <code>from_pk</code>, <code>after_pk</code> is converted to one <code>base</code> location value and options <code>with_base</code> and <code>by_pk</code>.